python-main.js

```diff
@@ -1069,6 +1086,7 @@ function web_editor(config) {
             reader.readAsText(file);
         }
         $('#editor').focus();
+        return ext
```

Comment:
Rather than return the extension from `doDrop()` which might a very clear return value, what if we just called `invalidExt()` from within this function?

Reply:
I thought this was the easiest way to close the Load modal before opening the warning modal. Should I call `invalidExt()` inside `doDrop()` and then close the Load modal inside `invalidExt()`?

Reply:
Perhaps the modalMsg() function should close any currently opened modal, that way we could call it from within doDrop.